Pythonwin/pywin/user.cfg

```diff
@@ -0,0 +1,100 @@
+# This is the user keyboard configuration file (user.cfg) for Pythonwin - for
```

Comment:
Is this file accidentally checked in? Maybe we could check a file in with the comments and add the filename to .gitignore?